index.js

```diff
@@ -1,10 +1,114 @@
 'use strict';
 /* eslint-env node */
-var SassCompilerFactory = require('broccoli-sass-source-maps');
-var path = require('path');
-var VersionChecker = require('ember-cli-version-checker');
-var Funnel = require('broccoli-funnel');
-var mergeTrees = require('broccoli-merge-trees');
+
+const path = require('path');
+const Filter = require('broccoli-persistent-filter');
```

Comment:
The use of broccoli-persistent-filter here seems odd to me. As far as I can tell, this is **not** a 1:1 compiler (which is essentially what `broccoli-persistent-filter` expects to be working with).

Reply:
what would be a good way to get this type of caching without broccoli-persistent-filter?